data/core/encyclopedia/geography.cfg

```diff
@@ -138,7 +138,7 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
 [topic]
     id=northlands
     title= _ "Northlands"
-    text= _ "There is no government over the freezing Northlands, only unstable clans in endless conflict. It is an unlawful and perilous place plagued by both its cold, icy terrain and the barbarians inhabiting it. Various groups of orcs, dwarves, men, gryphons, ogres, and even elves populate the region. Yet despite the sheer wilderness above the far north, the subterranean Dwarvish Kingdoms of Knagla and Kal Kartha are exceptions to their anarchic surroudings, providing both heat and protection to their people.While they may often be in conflict with orcs and trolls, the heart of Dwarvish territory is considered relatively safe in comparison. It is ruled by a functioning and usually, stable government. The northern and eastern borders are not defined, the southern border is the <ref>dst='great_river' text='Great River'</ref>, and the western border is the <ref>dst='great_ocean' text='ocean'</ref>.
+    text= _ "There is no government over the freezing Northlands, only unstable clans in endless conflict. It is an unlawful and perilous place plagued both by its cold, icy terrain and the barbarians inhabiting it. Various groups of orcs, dwarves, men, gryphons, ogres, and even elves populate the region. Yet despite the sheer wilderness above the far north, the subterranean Dwarvish Kingdoms of Knagla and Kal Kartha are exceptions to their anarchic surroudings, providing both heat and protection to their people.While they may often be in conflict with orcs and trolls, the heart of Dwarvish territory is considered relatively safe in comparison. It is ruled by a functioning and usually, stable government. The northern and eastern borders are not defined, the southern border is the <ref>dst='great_river' text='Great River'</ref>, and the western border is the <ref>dst='great_ocean' text='ocean'</ref>.
```

Comment:
This change is downright incorrect. The construction "**both** … **and** …" is a correlative conjunction, which means its two arguments must be of the same grammatical class. Currently, what you have is like this:

> **both** by its cold, icy terrain — _a prepositional phrase_
> **and** the barbarians inhabiting it — _a noun phrase_

The two arguments have different classes – a prepositional phrase vs a noun phrase. Now look at how it was previously:

> by
> **both** its cold, icy terrain — _a noun phrase_
> **and** the barbarians inhabiting it — _a noun phrase)

Here, both arguments have the same class (a noun phrase), and the _entire construct_ is the argument of the **by** preposition, making this a prepositional phrase.

The other option would be to duplicate **by**:

> **both** by its cold, icy terrain — _a prepositional phrase_
> **and** by the barbarians inhabiting it — _a prepositional phrase_

Here again, both arguments are of the same class (a prepositional phrase). Technically, using different prepositions on each side of the **and** would also work, though it doesn't fit in this specific case.